pyintegration/deephaven2/plot/figure.py

```diff
@@ -135,22 +135,22 @@ def axes(
         j_figure = self.j_figure
 
         if {"axes"}.issubset(non_null_args):
```

Comment:
Is there an advantage to doing it this way as opposed to:

```suggestion
        if "axes" in non_null_args:
```

Reply:
The code generation is more simple with the existing case since it properly handles multi-argument cases.  `in` may be faster, but I doubt it makes a practical difference for these use cases, where the plot rendering time is the slow part.  I'd just delay any change like that until we can justify more complexity for a needed speed improvement.